fabtests/common/shared.c

```diff
@@ -899,7 +899,7 @@ int ft_getinfo(struct fi_info *hints, struct fi_info **info)
 {
 	char *node, *service;
 	uint64_t flags = 0;
-	int ret;
+	int ret, efa_dev_idx;
```

Comment:
This is common code and should not have a provider specific indexing scheme.  Please find an API way of selecting a device